app/models/concerns/bulkrax/import_behavior.rb

```diff
@@ -9,7 +9,7 @@ def build_for_importer
         build_metadata
         raise CollectionsCreatedError unless collections_created?
         @item = factory.run
-      rescue RSolr::Error::Http => e
+      rescue RSolr::Error::Http, CollectionsCreatedError => e
```

Comment:
@geekscruff Should this be `Bulkrax::CollectionsCreatedError`, similar to https://github.com/samvera-labs/bulkrax/pull/116/files#diff-4d6669f1402d7a1df9781a44b4f9d438R18?

Reply:
I would have thought so, I'll do some testing in case I did that for a reason.